### PR TITLE
Feat/299b home page styling 

### LINKED ISSUE #299: style(home-page): improve home page styling
### Suggestions from Dec 5 meeting

- [x] Change title font to Sophia
- [x] Maybe add transparent black line across entire picture
- [x] For titles, options:
    - [ ] instead of rectangle, make the right end into a flag
    - [x] or round the corners like how the recipe cards are rounded
- [x] Move categories to left align with the first recipe card in the grid and make it similiar design to GitHub Issues
- [x] Remove About Us and shrink team logo and add to Footer
- [ ] Ensure that recipes in grid are not duplicated
- [x] Add padding to bottom between content and footer

### PR DESCRIPTION
**Pull request by edbertdai**
Added padding between last category and footer so card hover does not overlap footer.
Made header title text opaque so it is more readable.

Closes #299 again

### CODE REVIEW

File: source/components/home-page/home-page.html [hunk 1]
```diff
@@ -43,3 +43,5 @@ <h2 id="recipe-category-5" class="recipe-category">Gluten-Free</h2>
   <i class="fa fa-angle-right arrow-button-right" id="next-button-5"></i>
   <div id="recipe-card-grid-5" class="recipe-card-grid"></div>
 </div>
+
+<div class="bottom-padding"></div>
```
Comment (jbutera94) [inline source/components/home-page/home-page.html:47]: Do we need a separate `<div>` for this? Maybe instead we can wrap the recipe card sections in a div with `id="recipe-card-section-container"` and put margin bottom on that?

Comment (jbutera94) [reply source/components/home-page/home-page.html:47]: Another option would be to use `:last-child` on the `.recipe-card-section` class and put margin bottom on that.

**Review by jbutera94 — CHANGES_REQUESTED** [review]
Looks good, left a small suggestion for the padding at the bottom

File: source/components/home-page/home-page.html (hunk 1 above)
Comment (edbertdai) [reply source/components/home-page/home-page.html:47]: Yeah that's a good idea

**Review by jbutera94 — APPROVED** [review]

**Pull request merged**